utils/src/main/java/software/amazon/awssdk/utils/Validate.java

```diff
@@ -630,6 +630,18 @@ public static Duration isPositive(Duration duration, String fieldName) {
         return duration;
     }
 
+    public static Duration isPositiveIfPresent(Duration duration, String fieldName) {
+        if (duration == null) {
+            return null;
+        }
```

Comment:
Maybe `isPositiveOrNull`? "Present" implies there's an optional involved, to me. Can `isPositive` delegate to here now for its positivity check?

I also think the naming convention elsewhere is to include "param" in the method name if the second parameter is a field name. Otherwise it would be a string format for the caller to specify the whole message... but it looks like that's already been broken by `isPositive`, so nevermind!